sonar-project.properties

```diff
@@ -0,0 +1,3 @@
+sonar.organization=tbarlow12
+sonar.projectKey=tbarlow12_projector
```

Comment:
Should this be checked in?

Reply:
for posterity - not secrets. sonar needs this file at the root to run analysis